src/main/java/org/jabref/gui/entryeditor/fileannotationtab/FileAnnotationTabViewModel.java

```diff
@@ -84,7 +89,11 @@ public void notifyNewSelectedFile(Path newFile) {
                 .map(FileAnnotationViewModel::new)
                 .collect(Collectors.toList());
         annotations.setAll(newAnnotations);
-
+        if (annotations.isEmpty()) {
```

Comment:
`annotationEmpty = annotations.emptyProperty()` should also work and is a bit simpler.